libs/SalesforceSDKCommon/SalesforceSDKCommonTests/KeychainItemManagerTests.swift

```diff
@@ -72,138 +68,124 @@ final class KeychainItemManagerTests: XCTestCase {
         }
     }
     
-    func testCreateIfNotPresentNilAccount() {
+    func testCreateMultipleTimes() {
+        let accountName = "account.two"
         let serviceName = "test.two"
-        let check = KeychainHelper.read(service: serviceName, account: nil)
-        XCTAssertFalse(check.success)
-        
-        let keychainResult = KeychainHelper.createIfNotPresent(service: serviceName, account: nil)
-        XCTAssertTrue(keychainResult.success)
-        let keychainReadResult = KeychainHelper.read(service: serviceName, account: nil)
-        XCTAssertTrue(keychainReadResult.success)
-        XCTAssertNil(keychainReadResult.data)
+        var readResult = KeychainHelper.read(service: serviceName, account: accountName)
+        XCTAssertFalse(readResult.success)
+        var creationResult = KeychainHelper.createIfNotPresent(service: serviceName, account: accountName)
+        XCTAssertTrue(creationResult.success)
         
-        let data = "ATESTSTRING2".data(using: .utf8)  ?? Data()
-        let writeResult = KeychainHelper.write(service: serviceName, data: data, account: nil)
-        XCTAssertTrue(writeResult.success)
-        XCTAssertNotNil(writeResult.data)
-        
-        let readResult = KeychainHelper.read(service: serviceName, account: nil)
+        readResult = KeychainHelper.read(service: serviceName, account: accountName)
         XCTAssertTrue(readResult.success)
-        XCTAssertNotNil(readResult.data)
-        
+        XCTAssertNil(readResult.data)
+        XCTAssertTrue(readResult.status == errSecSuccess)
```

Comment:
possible nit: is there a reason not to use XCTAssertEqual here and other places? I think the failure message would be more descriptive

Reply:
Good call! Done.